python/pyspark/sql/tests/connect/test_connect_function.py

```diff
@@ -14,17 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import os
```

Comment:
Seems that this should be kept but not removed?

Reply:
Yea, good catch.
I removed it just as per IDEA's suggestiong...